raiden/transfer/state_change.py

```diff
@@ -116,6 +116,18 @@ def channel_identifier(self) -> ChannelID:
         return self.canonical_identifier.channel_identifier
 
 
+@dataclass(frozen=True)
+class ActionChannelCoopSettle(StateChange):
+    """Withdraw funds from channel."""
```

Comment:
This comment is copy-pasted from `ActionChannelWithdraw` and needs updating.

Reply:
Done.